rust/private/common.bzl

```diff
@@ -55,6 +54,5 @@ rust_common = struct(
     crate_info = CrateInfo,
     dep_info = DepInfo,
     stdlib_info = StdLibInfo,
-    default_edition = DEFAULT_RUST_EDITION,
```

Comment:
Maybe it's worth leaving this for a little bit to reduce the impact to public API? I'm torn though.

Reply:
I don't have any strong answers. Just noticing we're deleting a value from `rust_common` and thought maybe there should be more comms about this. But it can also be a note in the next release.